interface/java_binding/src/cpp/KTXTexture2.cpp

```diff
@@ -27,6 +32,25 @@ extern "C" JNIEXPORT jint JNICALL Java_org_khronos_ktx_KTXTexture2_getSupercompr
     return get_ktx2_texture(env, thiz)->supercompressionScheme;
 }
 
+//extern "C" JNIEXPORT jint JNICALL Java_org_khronos_ktx_KTXTexture2_compressAstcEx(JNIEnv *env,
```

Comment:
@MarkCallow I added commented-out code for the ASTC compression (it compiles but fails to link). Whenever the ASTC compression is released out, you can uncomment it (here and in the `KTXTexture2` Java class)

Reply:
Ok, the ASTC encoder is now done in Java!